src/SignUp/SignUP.module.css

```diff
@@ -0,0 +1,219 @@
+.signuptxt {
+  color: black;
+  background-color: #f1eaeb;
+  padding: 8px 12px 8px 12px;
+  border-radius: 30px;
+  font-weight: 700;
+}
+.signuptxt:hover {
+  background-color: #d3cfd0;
+}
+
+.logo {
+  color: #e60023;
+  cursor: pointer;
+  font-weight: 700;
+  margin-top: -40px;
+}
+
+.headertxt {
+  font-size: 200%;
+  font-weight: 600;
+  margin-top: -15px;
+}
+
+.headertxt2 {
+  position: relative;
+  bottom: 35px;
+  font-size: 15px;
+  font-weight: 400;
+  color: rgb(131, 126, 126);
+}
+
+.logintxt {
+  color: #fff;
+  background-color: #e60023;
+  padding: 8px 12px 8px 12px;
+  border-radius: 30px;
+  font-weight: 700;
+}
+.logintxt:hover {
+  background-color: rgb(206, 46, 46);
+}
+
+.cancel {
+  font-weight: 1000;
+  font-size: 1000px;
+}
+
+.input {
```

Comment:
Same here, never ever add any css directly to the element property instead add/create/use class to add style properties